druid/src/env.rs

```diff
@@ -14,13 +14,13 @@
 
 //! An environment which is passed downward into the widget tree.
 
-use std::any;
 use std::borrow::Borrow;
 use std::collections::HashMap;
 use std::fmt::{Debug, Formatter};
 use std::marker::PhantomData;
 use std::ops::Deref;
 use std::sync::Arc;
+use std::{any, collections::hash_map::Entry};
```

Comment:
Sorry, can't help my self
```suggestion
use std::any;
use std::borrow::Borrow;
use std::collections::{HashMap, hash_map::Entry};
use std::fmt::{Debug, Formatter};
use std::marker::PhantomData;
use std::ops::Deref;
use std::sync::Arc;
```